packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -103,38 +81,49 @@ export default function WorkflowListPage({ workflows }: Props) {
-                              </span>
-                            )}
-                            <span>{t(`${workflow.trigger.toLowerCase()}_trigger`)}</span>
-                          </div>
+                              <span className="mr-1">{t("triggers")}</span>
+                              {workflow.timeUnit && workflow.time && (
+                                <span className="mr-1">
+                                  {t(`${workflow.timeUnit.toLowerCase()}`, { count: workflow.time })}
+                                </span>
+                              )}
+                              <span>{t(`${workflow.trigger.toLowerCase()}_trigger`)}</span>
+                            </div>
+                          </Badge>
                         </li>
-                        <li className="mb-1 flex items-center whitespace-nowrap rounded-sm bg-gray-100 px-1 py-px text-xs text-gray-800 dark:bg-gray-900 dark:text-white">
-                          {workflow.activeOn && workflow.activeOn.length > 0 ? (
-                            <Tooltip
-                              content={workflow.activeOn.map((activeOn, key) => (
-                                <p key={key}>{activeOn.eventType.title}</p>
-                              ))}>
+                        <li>
+                          <Badge variant="gray">
+                            {workflow.activeOn && workflow.activeOn.length > 0 ? (
+                              <Tooltip
+                                content={workflow.activeOn.map((activeOn, key) => (
+                                  <p key={key}>{activeOn.eventType.title}</p>
+                                ))}>
+                                <div>
+                                  <FiLink className="mr-1.5 inline h-3 w-3" aria-hidden="true" />
+                                  {t("active_on_event_types", { count: workflow.activeOn.length })}
+                                </div>
+                              </Tooltip>
+                            ) : (
                               <div>
                                 <FiLink className="mr-1.5 inline h-3 w-3" aria-hidden="true" />
-                                {t("active_on_event_types", { count: workflow.activeOn.length })}
+                                {t("no_active_event_types")}
                               </div>
-                            </Tooltip>
-                          ) : (
-                            <div>
-                              <FiLink className="mr-1.5 inline h-3 w-3" aria-hidden="true" />
-                              {t("no_active_event_types")}
-                            </div>
-                          )}
+                            )}
+                          </Badge>
                         </li>
+                        {workflow.teamId && (
+                          <li>
+                            <Badge variant="gray">
+                              <>{workflow.team?.name}</>
```

Comment:
I think we should use differnet variant for signifying the team.
Also, the best would be to show workflows categorized like we do for EventTypes.

Reply:
Agree! @Jaibles is still working on improving designs for this to have it consistent. For now, he said this works fine like that. So a follow-up PR will come for that once design is ready!